src/book/03-guides/03-hardware/02-flashing.mdx

```diff
@@ -163,3 +163,57 @@ The rest of these instructions assume a network using WPA2 (like the network use
      ```
    - Be sure to replace `<WiFi Interface>` with the name of the interface found earlier.
    - Wait a handful of seconds and run `ip addr` to check that the WiFi interface has an IP address
+
+
+## Switching Wifi Networks
+
+Switching wifi networks can be accomplished with the following two commands:
+
+```
+$ wpa_cli list_networks
+Selected interface 'wlp58s0'
+network id / ssid / bssid / flags
+0	epsilon-z	any	[CURRENT]
+1	epsilon-x	any	[DISABLED]
+2	NETGEAR89	any	[DISABLED]
+
+$ wpa_cli select_network 2
+Selected interface 'wlp58s0'
+client_loop: send disconnect: Broken pipe
```

Comment:
```suggestion
# wpa_cli list_networks
Selected interface 'wlp58s0'
network id / ssid / bssid / flags
0	epsilon-z	any	[CURRENT]
1	epsilon-x	any	[DISABLED]
2	NETGEAR89	any	[DISABLED]

# wpa_cli select_network 2
Selected interface 'wlp58s0'
client_loop: send disconnect: Broken pipe
```
Does this need to be run as root, or is this unnecessary on the robots?
It needs root on my systems and also the arch wiki documents that it needs root.

Reply:
Looks like [here](https://github.com/NUbots/NUbots/blob/d4d23617295fc66aac75864602fd08053f812614/doc/ArchInstall/arch-chroot_install.sh#L139) we set wheel to be able to change settings.